webapp/app/forms/steps/steuerlotse_step.py

```diff
@@ -85,6 +85,11 @@ def _override_session_data(self, stored_data, session_data_identifier=None):
     def _handle_redirects(self):
         if self.render_info.redirect_url:
             return redirect(self.render_info.redirect_url)
+    
+    # Override this function, to manipulate the overview value
+    @classmethod
+    def get_overview_value_representation(cls, value):
+        return value
```

Comment:
How does this work without also giving the `Step` from the old MultiStep world the same method?

Reply:
The MultiStepFlow should use `FormStep`. Did I miss something with the MultiStepFlow? Don't get it :(

Reply:
Ah okay, I overlooked that, thanks for explaining! :)